config/params.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LogLevel;
+
+return [
+    'aliases' => [
+        '@root' => dirname(__DIR__),
+        '@assets' => '@root/public/assets',
+        '@assetsUrl' => '/assets',
+        '@npm' => '@root/node_modules',
+        '@public' => '@root/public',
+        '@resources' => '@root/resources',
+        '@runtime' => '@root/runtime',
+        '@views' => '@root/resources/views'
+    ],
+
+    'yiisoft/app' => [
```

Comment:
I'd either use `app` or remove the prefix altogether since this is a top-level package.

Reply:
done.